src/wallet/walletdb.cpp

```diff
@@ -422,8 +422,17 @@ ReadKeyValue(CWallet* pwallet, CDataStream& ssKey, CDataStream& ssValue,
                 strType != "minversion" && strType != "acentry") {
             wss.m_unknown_records++;
         }
+    } catch (const std::exception& e)
```

Comment:
nit, `{` here.

Reply:
Fixed. It's annoying that the linter doesn't catch stuff like this. Then the wrong version can crop up in the code, and people like me can copy it. :-)